DIRECTORY.md

```diff
@@ -146,6 +146,7 @@
     * Number Theory
       * [Prime Numbers](https://github.com/TheAlgorithms/Python/blob/master/data_structures/hashing/number_theory/prime_numbers.py)
     * [Quadratic Probing](https://github.com/TheAlgorithms/Python/blob/master/data_structures/hashing/quadratic_probing.py)
+    * [Luhn's Algorithm](https://github.com/TheAlgorithms/Python/blob/master/data_structures/hashing/luhn.py)
```

Comment:
Don't update this file, this is generated in the CI pipeline

> Do not update the README.md or DIRECTORY.md file which will be periodically autogenerated by our Travis CI processes.

Reply:
my bad